test/ctia/store_test.clj

```diff
@@ -0,0 +1,67 @@
+(ns ctia.store-test
+  (:require [ctia.store :as sut]
+            [ctia.test-helpers.http :refer [app->APIHandlerServices]]
+            [ctia.test-helpers.fixtures :as fixt]
+            [ctia.test-helpers.core :as helpers :refer [GET POST PATCH]]
+            [ctim.examples.incidents :refer [incident-minimal]]
+            [ctia.store-service :as store-svc]
+            [clojure.test :refer [deftest testing is]]))
+
+(def admin-ident {:login "johndoe"
+            :groups ["Administators"]})
+
+(deftest list-all-pages-test
+  (testing "list all pages shall properly list all results for given entity and parameters. "
+    (helpers/fixture-ctia-with-app
+     (fn [app]
+       (let [{:keys [StoreService] :as services} (app->APIHandlerServices app)
+             incidents-1 (fixt/n-doc incident-minimal 10)
+             incidents-2 (->> (fixt/n-doc incident-minimal 10)
+                              (map #(assoc % :source "incident-2-source")))
+             incidents (concat incidents-1 incidents-2)
+             bulk-result (helpers/POST-bulk app {:incidents incidents})]
+         (Thread/sleep 1000) ;; ensure index refresh
```

Comment:
`wait_for` is not propagated to event creation, so wait for index refresh configured to "1s".

Reply:
I can see how this could fail. You will be safer to add 500ms.